src/chat/Conversation.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+export default class Conversation extends React.Component {
+  render () {
+    const currentContact = this.props.currentContact;
+    return (
+      <div id="conversation">
+        {
+          currentContact ?
```

Comment:
Pas besoin de ternaire, faire un `return null` si le `this.props.currentContact` est `null`